in-consideration.md

```diff
@@ -0,0 +1,15 @@
+|Wassies by Wassies|0x1d20a51f088492a0f1c57f047a9e30c9ab5c07ea|2021-09-22|12345|1 Day: 2.05<br/>7 Day: 30.3<br/>30 Day: 119.68<br/>All Time: 11104.78|7 day time weighted average top collection bid:0.41399<br/>Number of bids above 50% of 0.41399: 1574
+|MoonCats|0xc3f733ca98e0dad0386979eb96fb1722a1a05e69|2021-04-22|19523|1 Day: 55.18<br/>7 Day: 146.45<br/>30 Day: 330.24<br/>All Time: 20540.03|7 day time weighted average top collection bid:0.23839<br/>Number of bids above 50% of 0.23839: 1552
+|goblintown.wtf|0xbce3781ae7ca1a5e050bd9c4c77369867ebc307e|2022-05-19|9999|1 Day: 47.74<br/>7 Day: 593.03<br/>30 Day: 1903.89<br/>All Time: 65664.89|7 day time weighted average top collection bid:0.54659<br/>Number of bids above 50% of 0.54659: 220
+|Sappy Seals|0x364c828ee171616a39897688a831c2499ad972ec|2021-08-31|10000|1 Day: 275.62<br/>7 Day: 2436.77<br/>30 Day: 4454<br/>All Time: 22593.13|7 day time weighted average top collection bid:1.0472<br/>Number of bids above 50% of 1.0472: 3881
+|Moonbirds|0x23581767a106ae21c074b2276d25e5c3e136a68b|2022-04-15|10000|1 Day: 2937.75<br/>7 Day: 14179.7<br/>30 Day: 38302.85<br/>All Time: 286671.05|7 day time weighted average top collection bid:5.94782<br/>Number of bids above 50% of 5.94782: 1070
+|Chimpers|0x80336ad7a747236ef41f47ed2c7641828a480baa|2022-05-19|5555|1 Day: 93.76<br/>7 Day: 1110.84<br/>30 Day: 3211.5<br/>All Time: 21359.24|7 day time weighted average top collection bid:1.11929<br/>Number of bids above 50% of 1.11929: 2789
+|Finiliar|0x5a0121a0a21232ec0d024dab9017314509026480|2022-01-06|10000|1 Day: 10.52<br/>7 Day: 98.57<br/>30 Day: 320.96<br/>All Time: 4645.68|7 day time weighted average top collection bid:0.23691<br/>Number of bids above 50% of 0.23691: 169
+|World of Women|0xe785e82358879f061bc3dcac6f0444462d4b5330|2021-07-27|10000|1 Day: 33.1<br/>7 Day: 523.52<br/>30 Day: 1610.46<br/>All Time: 82420.14|7 day time weighted average top collection bid:1.79243<br/>Number of bids above 50% of 1.79243: 2019
+|Art Gobblers|0x60bb1e2aa1c9acafb4d34f71585d7e959f387769|2022-10-31T20:20:17.821Z|2986|1 Day: 1.2<br/>7 Day: 99.65<br/>30 Day: 1066.51<br/>All Time: 50107.24|7 day time weighted average top collection bid:0.87251<br/>Number of bids above 50% of 0.87251: 1390
+|Froggy Friends Official|0x29652c2e9d3656434bc8133c69258c8d05290f41|2022-03-11T07:12:49.521Z|4444|1 Day: 1.42<br/>7 Day: 24.22<br/>30 Day: 73.58<br/>All Time: 1263.22|7 day time weighted average top collection bid:0.10825<br/>Number of bids above 50% of 0.10825: 177
+|Doodles|0x8a90cab2b38dba80c64b7734e58ee1db38b8992e|2022-02-09T21:10:23.583Z|10000|1 Day: 2707.32<br/>7 Day: 9148.59<br/>30 Day: 36105.37<br/>All Time: 249731.11|7 day time weighted average top collection bid:5.21551<br/>Number of bids above 50% of 5.21551: 22809
```

Comment:
Doodles should probably not be in paprMEME, based on the fact that they are 
- already served by several other protocols providing instant loans 
- have a bid depth significantly above the other collections, suggesting lower risk
but they are included for consideration and discussion.